source/dotnet/Library/AdaptiveCards.Rendering.Wpf/AdaptiveRenderContext.cs

```diff
@@ -142,6 +145,9 @@ public SolidColorBrush GetColorBrush(string color)
             }
         }
 
+        // This dictionary exists for setting the ids to elements without messing with the Name property to avoid crashes with weird ids
+        public IDictionary<string, FrameworkElement> RendererElementsWithId = new Dictionary<string, FrameworkElement>();
```

Comment:
Do we need to make this public? I worry about future breaking API changes if we change this structure, like maybe we decide it's a dictionary of `<string, UIElement>` or something in the future.

I believe the only use cases for hosts using this is they need to toggle visibility of an element, which we already have a public toggle visibility method I believe?